examples/boiler/run.py

```diff
@@ -46,7 +50,7 @@ def from_numpy(cls, state: NDArray[np.float64]) -> Self:
         return cls(state[0])
 
 
-type TargetTemperature = float
+TargetTemperature = float
```

Comment:
```suggestion
TargetTemperature: TypeAlias = float
```
https://docs.python.org/3/library/typing.html#typing.TypeAlias